### PR TITLE
Show commands services

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
Please review, especially the command arguments and keywords.

### CODE REVIEW

File: docs/cli_reference.md [hunk 1]
```diff
@@ -7267,17 +7354,117 @@ show service-path [{service-name <name> | hierarchy-service-name <name> | contai
...
+               TestProbe1: Up
+               TestProbe2: Down
+
+```
+
+#### Version History
+
+| Release | Modification                |
+| ------- | ----------------------------|
+| 5.1.0   | Detail fields added: Next-hop, Peer, Path-Metrics, Index, Meets-SLA, Reachability-Probes |
```
Comment (lliu5374) [inline docs/cli_reference.md:7475]: same for the two fields here, which should only be in 5.2

**Review by lliu5374 — APPROVED** [review]

**Review by cmain — APPROVED** [review]

**Review by jacobbogdanov — CHANGES_REQUESTED** [review]

File: docs/cli_reference.md [hunk 2]
```diff
@@ -1363,6 +1367,27 @@ admin@labsystem1.fiedler#
 | 3.1.0   | The location of the exported configuration changed |
 
 
+## `generate config`
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/cli_reference.md]: changes made

File: docs/cli_reference.md [hunk 3]
```diff
@@ -4150,9 +4175,15 @@ Display configuration exports.
 #### Usage
 
 ```
-show config exports
+show config exports [<name>] [verbose] [flat]
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/cli_reference.md]: removed verbose and the explanation.

File: docs/cli_reference.md [hunk 4]
```diff
@@ -318,7 +318,7 @@ Display the differences between two configurations.
 #### Usage
 
 ```
-compare config [<old>] [<new>]
+compare config [<old>] [<new>] [<name>]
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/cli_reference.md]: removed the name argument and clarified.

**Review by jacobbogdanov — APPROVED** [review]

**Pull request merged**